<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employer</class>
 <widget class="QDialog" name="employer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1251</width>
     <height>701</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_cpemp">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>27</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_logout">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>27</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_employer">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>29</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Employer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jhlogo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_sj">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_pj">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Provide Job</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pj">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>180</y>
       <width>331</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_aj">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_vpj">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>View Provided Job</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vpj">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>181</y>
       <width>331</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_vp">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>361</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_vpemp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>View Profile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>341</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
